client/my-sites/site-settings/search.jsx

```diff
@@ -69,22 +71,56 @@ class Search extends Component {
 		);
 	}
 
-	renderSettingsContent() {
-		const { activatingSearchModule, searchModuleActive, translate } = this.props;
+	renderSettingsContent( updatingSettings, searchActive ) {
+		const { translate } = this.props;
 
 		return (
 			<div className="search__module-settings site-settings__child-settings">
-				{ activatingSearchModule && (
-					<FormSettingExplanation>{ translate( 'Activating search…' ) }</FormSettingExplanation>
+				{ updatingSettings && (
+					<FormSettingExplanation>{ translate( 'Updating settings…' ) }</FormSettingExplanation>
 				) }
-
-				{ searchModuleActive && <div>{ this.renderSearchExplanation() }</div> }
+				{ searchActive && ! updatingSettings ? (
+					<div>{ this.renderSearchExplanation() }</div>
+				) : null }
 			</div>
 		);
 	}
 
-	renderSettings() {
-		const { isRequestingSettings, isSavingSettings, siteId, translate } = this.props;
+	renderWPComSettings() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			fields,
+			handleAutosavingToggle,
+			translate,
+		} = this.props;
+
+		return (
+			<FormFieldset>
+				{ this.renderInfoLink( 'https://support.wordpress.com/jetpack-search/' ) }
```

Comment:
I'm getting an "Oops, nothing here" message when clicking this link. That seems fine if we haven't got the documentation complete yet.


<img width="779" alt="screen shot 2018-08-29 at 11 23 20 am" src="https://user-images.githubusercontent.com/1126811/44802242-d30fed00-ab80-11e8-91fe-d5a748ad5862.png">

Reply:
Ya I'm working on putting up the documentation